kaolin/io/obj.py

```diff
@@ -16,37 +16,45 @@
 import warnings
 from collections import namedtuple
 
-from PIL import Image
 import numpy as np
 import torch
+from PIL import Image
+
+ObjMesh = namedtuple('ObjMesh',
```

Comment:
Please don't rename the nametuple

Reply:
I think the string should also be renamed

Reply:
changed.